rhea/cores/fifo/fifo_fast.py

```diff
@@ -65,7 +65,7 @@ def fifo_fast(clock, reset, fbus, use_srl_prim=False):
         # the FIFO, only a read address is accounted.
         @always(clock.posedge)
         def rtl_srl_in():
-            if srlce:
+            if srlce and (not fbus.full):
```

Comment:
This change might be backed out later, the SRL fifo was organized so the SRL FIFO would be inferred, I don't think these extra checks can be used in the SRL fifo, the normal synchronous FIFO can have the additional checks.